Sources/ContentsquareInstance.swift

```diff
@@ -70,6 +103,19 @@ public class ContentsquareInstance: ContentsquareCommand {
     public func optOut() {
         Contentsquare.optOut()
     }
+    
+    private func parseIndex(_ value: Any?) -> UInt32? {
+        switch value {
+        case let i as UInt32:
+            return i
+        case let i as Int where i >= 0:
+            return UInt32(i)
```

Comment:
This can crash in case i is greater than `UInt32.max`.
You can use a safer check like so:

```suggestion
    private func parseIndex(_ value: Any?) -> UInt32? {
        switch value {
        case let i as any BinaryInteger:
            return UInt32(exactly: i)
```